### ISSUE TITLE
Question regarding channel ID vs webhook url

### ISSUE THREAD

**Issue by ghost** [arhill05/backstage-plugin-scaffolder-backend-module-slack#2]
I am investigating using this in my template flows.  I wondered about a small change:

Could we use the [slack_sdk]() like above - and then if there are multiple templates - there could be a single token/app and the user could say which channel to post to?

```
const { WebClient } = require('@slack/web-api');

// An access token (from your Slack app or custom integration - xoxp, xoxb)
const token = process.env.SLACK_TOKEN;

const web = new WebClient(token);

// This argument can be a channel ID, a DM ID, a MPDM ID, or a group ID
const conversationId = 'C1232456';

(async () => {
  // See: https://api.slack.com/methods/chat.postMessage
  const res = await web.chat.postMessage({ channel: conversationId, text: 'Hello there' });

  // `res` contains information about the posted message
  console.log('Message sent: ', res.ts);
})();
```

Does not look like it would be too hard to wire into get channel_id from `ctx` and slackToken from config?
Happy to investigate but thought I would ask before hand?

**Comment by arhill05** [comment]
Hey @dlaird-ovo! Thanks for the suggestion. I like this idea a lot, I'm just not familiar with the Slack SDK. 

I will put this on my todo list and take a look, but with it being the holiday season I might not be able to get to it until after the first of the year.

**Comment by arhill05** [comment]
Hey @dlaird-ovo! Happy new year!

I've published version 2.0.0 of this package that allows for sending a message either via a conversation ID or a conversation name. It requires you to create a Slack App and grant it the proper OAuth scopes in order to get a token, but that is documented in the README. 

Thanks for the suggestion, and let me know if this solves what you were looking for!

**Comment by ghost** [comment]
Will give that a go as soon as I can and come back with feedback .  Thankyou
